src/main/java/net/datafaker/transformations/sql/SqlTransformer.java

```diff
@@ -484,7 +484,6 @@ public String getValue(Case caze) {
                 case UPPERCASE -> upperCaseValue;
                 case LOWERCASE -> lowerCaseValue;
                 case CAPITAL -> capitalValue;
-                default -> throw new IllegalArgumentException("Unknown case " + caze);
```

Comment:
what happens if someone adds a new value to enum and forgets to update this `switch`?
With `default` exception the problem could be detected immediately. 
How to detect it without `default`?

Reply:
Then it won't compile:

<img width="637" alt="image" src="https://user-images.githubusercontent.com/46468/218306565-756d57be-3e3b-4e83-ad9e-81bc459dcdf0.png">

Reply:
ok then